designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -1340,10 +1343,28 @@ internal fun Background.asBrush(
         }
         is Background.Image -> {
             val backgroundImage = this
+            val imageTransform = backgroundImage.transform.asSkiaMatrix()
+            backgroundImage.res_name.orElse(null)?.let {
+                val resId =
+                    appContext.resources.getIdentifier(it, "drawable", appContext.packageName)
+                if (resId != Resources.ID_NULL) {
+                    return Pair(
+                        RelativeImageFill(
+                            image = BitmapFactory.decodeResource(appContext.resources, resId),
```

Comment:
I'd like to know if this decode function does any kind of caching, because it looks like this could get called over and over again and be quite slow.

Reply:
I will look into it in a separate pr. Yes it will be called every time draw triggers. I will do caching in a following pr.